Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
I agree that html5ever should do one thing and do it well. Ideally, even Kuchiki should be orthogonal to how you get a `std::io::Read` (such as `hyper::client::Response`) to give to it. The thing is that getting a `charset` parameter of a `Content-Type` header is non-obvious, and users might not even think of doing it.

@nox, what do you think? Should we move Hyper support back from html5ever?